modules/ui/src/main/scala/lucuma/ui/primereact/FormInputText.scala

```diff
@@ -50,23 +50,15 @@ object FormInputText {
   private val component = ScalaFnComponent[FormInputText] { props =>
     val sizeCls = props.size.toOption.map(_.cls).orEmpty
 
-    def buildAddons(addons: List[TagMod | CButton.Builder]) =
```

Comment:
Moved to an extension method in an unrelated change. It turns out that (undocumented) dropdowns support addons, too. So, at some point I'll try to make changes for that...